core/templates/dev/head/pages/topic_editor/questions/QuestionsTabDirective.js

```diff
@@ -118,6 +125,117 @@ oppia.directive('questionsTab', [
             });
           };
 
+          $scope.getSuggestedQuestions = function() {
+            $scope.questionSuggestionThreads = [];
+            $http.get('/generalsuggestionlisthandler', {
```

Comment:
Look into $q.all(). Don't fire one after the other, do them together.

Reply:
Done